README.md

```diff
@@ -114,7 +118,7 @@ export AWS_SECRET_ACCESS_KEY=bar
 ```
 
 ### Add the SSH Key to the SSH Agent
-You will need to add the private key to your SSH agent: 
+Create your EC2 Key Pair with name "default" in the AWS Region you picked.  You will need to add the private key to your SSH agent: 
```

Comment:
The keypair does not have to be default. You can specify the keypair to use. Please refactor.